pulp_ansible/app/tasks/collections.py

```diff
@@ -793,14 +828,31 @@ async def _download_unpaginated_metadata(self):
 
             collection_endpoint = f"{root_endpoint}/collections/all/"
             excludes_endpoint = f"{root_endpoint}/excludes/"
+            namespaces_endpoint = f"{root_endpoint}/namespaces/?paginate=false"
```

Comment:
This needs to work with older galaxy_ng servers. galaxy_ng already has a `/namespaces/` api endpoint that breaks when the sync requests it.

I don't think you need to worry about making namespace sync work on older versions of galaxy_ng, but it should at the very least not break

Reply:
where does this paginate=false flag come from? that seems like a very dangerous thing to have on the API from a denial of service perspective

Reply:
Good point! I didn't think of the older versions, I'll make changes like you suggested below and I'll remove the `paginate` flag from the viewset.